### PR TITLE
NAS-102955 doc rsync path limit being 1023 characters.

### PR DESCRIPTION
**Pull request by a-stjohn**

### CODE REVIEW

File: userguide/snippets/pathlengths.rst [hunk 1]
```diff
@@ -19,16 +19,16 @@ can also reduce the length of these limits by one or more bytes.
...
    |                     |                | file itself. For example, the path                                     |
    |                     |                | :file:`/mnt/tank/mydataset/mydirectory/myfile.txt` is 42 bytes long.   |
    |                     |                |                                                                        |
-   |                     |                | Using very long file or directory names can be problematic. A complete |
-   |                     |                | path with long directory and file names can exceed the 1024-byte       |
-   |                     |                | limit, preventing direct access to that file until the directory names |
-   |                     |                | or filename are shortened or the file is moved into a directory with a |
-   |                     |                | shorter total path length.                                             |
+   |                     |                | Using very long file or directory names can be problematic. If a       |
+   |                     |                | path with long directory and file names exceed the 1023-byte           |
```
Comment (Mrt134) [inline userguide/snippets/pathlengths.rst]:
```suggestion
   |                     |                | path with long directory and file names exceeds the 1023-byte          |
```

File: userguide/tasks.rst [hunk 2]
```diff
@@ -288,8 +288,8 @@ task.
    |                                  |                             |                                                                                           |
    |                                  |                             |                                                                                           |
    +==================================+=============================+===========================================================================================+
-   | Path                             | browse button               | :guilabel:`Browse` to the path to be copied. Path lengths cannot be greater               |
-   |                                  |                             | than 255 characters.                                                                      |
+   | Path                             | browse button               | :guilabel:`Browse` to the path to be copied. Follows the FreeBSD                          |
```
Comment (wblock) [inline userguide/tasks.rst]: It's not really "follows".  (Fix whitespace.)
```suggestion
   | Path                             | browse button               | :guilabel:`Browse` to the path to be copied. The FreeBSD                          |
```

File: userguide/tasks.rst [hunk 3]
```diff
@@ -288,8 +288,8 @@ task.
...
+   |                                  |                             | :ref:`path and name length <Path and Name Lengths>` standards.                            |
```
Comment (wblock) [inline userguide/tasks.rst]: (Fix whitespace.)
```suggestion
   |                                  |                             | :ref:`file path limits <Path and Name Lengths>` apply.                            |
```

File: userguide/intro.rst [hunk 4]
```diff
@@ -279,6 +279,9 @@ These screen options have changed:
 * A :guilabel:`Last Snapshot` column has been added to
   :ref:`Replication Tasks`.
 
+* :guilabel:`Path` in :ref:`rsync task <Rsync Tasks>` abides by the
```
Comment (wblock) [inline userguide/intro.rst]: As below, it's not a rule, it's a limitation.

**Pull request merged**